test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +44,33 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
good combinatorial approach! but there something else you want to test too. think deep